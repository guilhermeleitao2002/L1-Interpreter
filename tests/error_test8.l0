let add = fn x, y => { x + y };
add(1, 2);;