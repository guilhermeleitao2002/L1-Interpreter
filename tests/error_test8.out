Syntax Error:
ParseException: Encountered " "," ", "" at line 2, column 6.
Was expecting one of:
    "+" ...
    "-" ...
    "*" ...
    "/" ...
    "(" ...
    ")" ...
    "&&" ...
    "||" ...
    "==" ...
    ">" ...
    "<" ...
    ">=" ...
    "<=" ...
    "~=" ...
    "::" ...
    "(" ...
    